tests/test_dnatracing.py

```diff
@@ -0,0 +1,63 @@
+"""Tests of dnatracing methods."""
+import numpy as np
+import pytest
+
+
+def test_get_grain_array(test_dnatracing) -> None:
+    """Test extraction of grain array."""
+    TARGET = {
```

Comment:
Should this be a fixture itself? Maybe not.

Reply:
Think this is the same as comment above, as these are unit-tests testing methods in isolation then I don't envisage needing them to be fixtures. If fixtures are needed I'll define them in `tests/conftest.py`.